BedrockServer.cpp

```diff
@@ -1576,8 +1576,10 @@ void BedrockServer::_reply(unique_ptr<BedrockCommand>& command) {
     // Finalize timing info even for commands we won't respond to (this makes this data available in logs).
     command->finalizeTimingInfo();
 
-    // Don't reply to commands with pseudo-clients (i.e., commands that we generated by other commands).
+    // Don't reply to commands with pseudo-clients (i.e., commands that we generated by other commands, or using
+    // `Connection: forget`.
```

Comment:
```suggestion
    // `Connection: forget`)
```

(but also I'm not a fan of breaking a line for 20 chars which can fit on modern screens)

Reply:
My editor auto line breaks at 120 chars. Are we not doing that anymore? I can turn it off, it's done that for like 15 years.

Reply:
Heh, we had a discussion about this somewhere some day. https://expensify.slack.com/archives/C03TQ48KC/p1627427210244000
https://expensify.slack.com/archives/C03TQ48KC/p1627495279311100

Soooo I guess you can make it 190? 😄